src/Main.elm

```diff
@@ -705,16 +713,24 @@ viewFormFieldOptionsPreview { formValues, customAttrs, shortTextTypeDict } formF
                     False
     in
     case formField.type_ of
-        ShortText inputType maybeMaxLength ->
+        ShortText inputType attrs ->
             let
                 shortTextAttrs =
                     Dict.get inputType shortTextTypeDict
                         |> Maybe.withDefault Dict.empty
                         |> Dict.toList
                         |> List.map (\( k, v ) -> attribute k v)
 
+                attrsDict =
+                    Dict.fromList attrs
+
                 extraAttrs =
-                    [ Maybe.map (\maxLength -> maxlength maxLength) maybeMaxLength
+                    [ Dict.get "maxlength" attrsDict
+                        |> Maybe.andThen String.toInt
+                        |> Maybe.map (\v -> maxlength v)
+                    , Dict.get "mulitple" attrsDict
+                        |> Maybe.andThen Bool.Extra.fromString
+                        |> Maybe.map (\v -> multiple v)
```

Comment:
we can convert `List (String, String)` to `List (Html.Attribute msg)` using https://package.elm-lang.org/packages/elm/html/latest/Html-Attributes#attribute 

no need to use `String.toInt` `maxlength` or `Bool.Extra.fromString` `multiple`

---

also there's a typo ~"mulitple"~ `"multiple"` uncaught in tests. maybe our [`MainTest.inputFieldFuzzer`](https://github.com/choonkeat/tiny-form-fields/blob/9b0e4d84443b7fef6a8b02987af7760fd6875cab/tests/MainTest.elm#L98-L102) should not ONLY rely on `allInputField` (which is used in the app dropdown ui)

```diff
    inputFieldFuzzer : Fuzzer InputField
    inputFieldFuzzer =
+       allInputField ++ moreTestInputFields
            |> List.map Fuzz.constant
            |> Fuzz.oneOf

+ moreTestInputFields : List InputField
+ moreTestInputFields =
+       [ ....
```

then we would've caught this typo